taxi/views.py

```diff
@@ -34,6 +35,19 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    fields = "__all__"
```

Comment:
Ensure that using `fields = "__all__"` is intentional and that all fields of the `Manufacturer` model should be editable by users. If there are fields that should not be exposed, consider specifying only the fields that should be editable.